app/TuistApp/Sources/Modifiers/MenuItemStyle.swift

```diff
@@ -17,3 +16,11 @@ extension View {
         modifier(MenuItemSyle())
     }
 }
+
+struct MenuItemButtonStyle: ButtonStyle {
```

Comment:
```suggestion
private struct MenuItemButtonStyle: ButtonStyle {
```

Let's make it private to this file